test_suite.py

```diff
@@ -80,26 +86,30 @@ def test_ownership3_4(child_sub, samples_sent, timeout, verbosity):
 
     return ReturnCode.RECEIVING_FROM_ONE
 
-# It checks if the Subscriber is receiving the samples in order.
-# It does <max_samples_received> iterations and at each of them it
-# will process one sample (taken from pexpect). For each sample it
-# checks if the sample received matches with the sample sent (taken
-# from <samples_sent[0]). In case if does not match it means the
-# subscriber is not receiving the samples in order, and it will
-# save the code "DATA_NOT_CORRECT", stopping the function.
-# In case it matches it will receive another sample and it will
-# continue with another iteration.
+
 def test_reliability_4(child_sub, samples_sent, timeout, verbosity):
+
+    """
+    The functions testS reliability, checking if the Subscriber receives the
+    samples in order.
+
+    child_sub: child program generated with pexpect
+    samples_sent: list of multiprocessing Queues with the samples
+                the Publishers send. Element 1 of the list is for
+                Publisher 1, etc.
+    timeout: time pexpect waits until it matches a pattern.
+    verbosity: print debug information.
+    """
     max_samples_received = 3
     max_wait_time = 5
     for x in range(0, max_samples_received, 1):
         # we take the numbers that identify the sample
```

Comment:
```suggestion
        # take the numbers that identify the sample
```

numbers? do we refer here to received or sent values?